src/encoded/commands/deploy.py

```diff
@@ -1012,6 +1016,8 @@ def hostname(value):
     parser.add_argument('--cluster-size', type=int, default=5, help="Elasticsearch cluster size")
     parser.add_argument('--es-ip', default='localhost', help="ES Master ip address")
     parser.add_argument('--es-port', default='9201', help="ES Master ip port")
+    parser.add_argument('--primary-shards', default=3, type=int, help="ES number of primary shards")
```

Comment:
I guess I'm still concerned that we're not going to have any replicates in production anymore?

Reply:
We have one replicate for each primary shard. Primary and replicate will be hosted in different nodes, and does provide some redundancy in case one goes down.